app/views/shared/_footer2.html.erb

```diff
@@ -1,42 +1,41 @@
-<footer class="bg-dark d-flex flex-row justify-content-evenly align-items-start pt-5 pb-5" >
+<footer class="footer2_section" >
   
   <div class="slogan_section">
     <div>
       <%= image_tag('jade_logo.png', class:'logo')%>
     </div>
     <div class="slogan">
-      <h4>SEARCH. BUILD. ENJOY.</h4>
+      <h2>SEARCH. BUILD. ENJOY.</h2>
     </div>
   </div>
 
   <div class="info_section">
     <div class="info">
-      <h5>CATEGORIES</h5>
+      <h3>CATEGORIES</h3>
       <ul class="info_items">
-        <li><a class="text-white" href="#">Build a PC</a></li>
-        <li><a class="text-white" href="#">Build Guide</a></li>
-        <li><a class="text-white" href="#">Completed Builds</a></li>
-        <li><a class="text-white" href="#">Browse Products</a></li>
+        <li><a class="info_text" href="#">Build a PC</a></li>
+        <li><a class="info_text" href="#">Build Guide</a></li>
+        <li><a class="info_text" href="#">Builds Gallery</a></li>
+        <li><a class="info_text" href="#">Browse Products</a></li>
       </ul>
     </div>
 
     <div class="info">
-      <h5>INFORMARION</h5>
+      <h3>INFORMARION</h3>
```

Comment:
Typo on "INFORMATION"

Reply:
alright edited the commit of _footer2.html.erb to correct the spelling error.